content/lua/concepts/mathematical-library/terms/tointeger/tointeger.md

```diff
@@ -0,0 +1,69 @@
+---
+Title: 'tointeger()'
+Description: 'Used to convert a given value to an integer if possible.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Math'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+In Lua, the **`math.tointeger()`** function is a standard function that is part of the Lua `math` library. It is used to convert a given value to an integer, if the given value _is convertible_ to an integer.
+
+> **Note:** `math.tointeger()` was introduced in Lua 5.3. Earlier verisons of Lua will not have this function as a standard part of their `math` library.
+
+## Syntax
+
+As `tointeger()` is a method that is a part of the standard Lua `math` library, it must be called as `math.tointeger()`.
+
+```pseudo
+math.tointeger(mycoolvalue) -- Returns an integer, if mycoolvalue is convertible to an integer. Otherwise returns nil.
+```
+
```

Comment:
````suggestion
math.tointeger(mycoolvalue)
```

`math.tointeger()` returns an integer, if `mycoolvalue` is convertible to an integer. Otherwise returns nil.
````

Prefer the explanatory text outside the code block.